WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
 }
 
 .ChooserDialog .DwtInputField {
-	padding-bottom:3px;
-	padding-top:3px;
+	@ChooseFolderDialogInputContainer@
 }
 
-.ChooserDialog INPUT {
-	width:100%;
+#ChooseFolderDialog .ChooserDialog INPUT {
+	@ChooseFolderDialogInputField@
 }
 
 .ChooserDialog .overview {
-	background-color:white;
-	border:1px solid black;
-	overflow:auto;
+	@ChooseFolderDialogFolderContainer@
```

Comment:
In general we should be targeting something more specific css selector if the style is specific for chooseFolder dialog, can use .ChooserDialog if its general for all dialogs using it.